### DISCUSSION TITLE
Dúvidas Profile Card Challenge

### DISCUSSION THREAD

**Discussion by rodolfobertini** [dcTeam23/course#9] [Q&A]
Na tarefa 
https://github.com/dcTeam23/profile_card_challenge/
O que exatamente tem que fazer? Eu me embananei todo e pensei q fosse pra refazer na mão o profile card component... consegui fazer quase tudo, menos deixar os 3 arquivos SVG com aquele efeito degradê ativo... depois de horas fazendo... ai vejo que tem um arquivo já do figma de modelo com tudo já feito e pronto.

**Comment by jv-farias** [comment]
Boa tarde, Rodolfo a atividade da semana é **transformar** esse arquivo que o professor disponibilizou tanto em Figma, quanto em JPG **em código (Html e Css)!**
O arquivo no Figma vai nos ajudar bastante para termos noções, por exemplo, dos espaçamentos entre cada texto, o tamanho da foto de perfil e até mesmo a largura da borda ao redor da imagem.

Espero ter ajudado!!

**Reply by rodolfobertini** [reply]
Pois é.... Eu acabei acordando umas 4h da manhã no dia q foi colocado o desafio, e li rapidamente, não vi nem o arquivo figma.... ai pensei q fosse pra recriar no figma aquele projeto hehehehehhe ai fui fazendo no figma... que inclusive achei uma ótima ferramenta... ai deposi lendo com calma vi realmente o que tem que ser feito... vou já começar :) obrigado pela explicação.

**Reply by jv-farias** [reply]
Aaah, tranquilo!! Qualquer dúvida estamos aqui!
